rotkehlchen/globaldb/schema.py

```diff
@@ -10,6 +10,18 @@
 );
 """
 
+DB_CREATE_RPC_NODES = """
+CREATE TABLE IF NOT EXISTS rpc_nodes (
```

Comment:
name it default rpc nodes and also write in a comment that these are the defaults for each new user